packages/authentication/src/__tests__/unit/providers/authentication.provider.unit.ts

```diff
@@ -3,116 +3,116 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Context, instantiateClass} from '@loopback/context';
-import {Request} from '@loopback/rest';
-import {expect} from '@loopback/testlab';
-import {Strategy} from 'passport';
-import {AuthenticateFn, AuthenticationBindings, UserProfile} from '../../..';
-import {AuthenticateActionProvider} from '../../../providers';
-import {MockStrategy} from '../fixtures/mock-strategy';
+// import {Context, instantiateClass} from '@loopback/context';
```

Comment:
@jannyHou stated I should comment out these tests for the meantime. Her PR will re-write the test.